snooty/types.py

```diff
@@ -190,17 +190,27 @@ def __contains__(self, key: str) -> bool:
 
         return False
 
-    def get_url(self, key: str) -> str:
-        # TODO: add check for local definitions here
+    def get_url(self, key: str) -> Tuple[str, str]:
+        # Check to see if the target is defined locally
+        try:
+            return ("fileid", self.local_definitions[key][0].as_posix())
```

Comment:
Wondering if there is any benefit to including the file suffix in the target node? If not, we could use FileId's [`without_known_suffix()`](https://github.com/mongodb/snooty-parser/blob/master/snooty/types.py#L47-L51) method.

Reply:
I don't see any need for the file suffix; happy to remove it!